### ISSUE TITLE
[steamshovel] Deprecation warnings after upgrading to IPython 4 (Trac #1359)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1351]
There was a big split in IPython between version 3 and 4; a lot of functionality is now part of Jupyter. When upgrading to IPython 4, steamshovel triggers deprecation warnings: the `IPython.qt` package should be replaced with `qtconsole`.  

Migrated from https://code.icecube.wisc.edu/ticket/1359
```json
{
    "status": "closed", 
    "changetime": "2015-09-21T18:46:58", 
    "description": "There was a big split in IPython between version 3 and 4; a lot of functionality is now part of Jupyter. When upgrading to IPython 4, steamshovel triggers deprecation warnings: the `IPython.qt` package should be replaced with `qtconsole`.  ", 
    "reporter": "kkrings", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1442861218590071", 
    "component": "combo core", 
    "summary": "[steamshovel] Deprecation warnings after upgrading to IPython 4", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-09-21T14:15:05", 
    "milestone": "", 
    "owner": "hdembinski", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150921T15:14:53: 
* hdembinski changed status from "new" to "accepted"
* hdembinski changed type from "defect" to "enhancement"

**Comment by icecube-trac** [comment]
Trac update at 20150921T15:21:09: hdembinski commented:

> Hi Kai, please try the latest trunk and tell me whether the fix makes the warnings disappear.

**Comment by icecube-trac** [comment]
Trac update at 20150921T15:28:49: kkrings commented:

> Replying to [comment:2 hdembinski]:
> > Hi Kai, please try the latest trunk and tell me whether the fix makes the warnings disappear.
> >> Hi Hans, there is another `IPython.qt` import in console_widget.py:52 and `RichIPythonWidget` is now `RichJupyterWidget`.

**Comment by icecube-trac** [comment]
Trac update at 20150921T16:55:18: hdembinski commented:

> Since you apparently know your way around in the code :) and you have the platform to test this, could you please try to make the patch work for you and then commit the changes? For me to debug this, I would have to install IPython4, and I would like to stick to the officially supported environment.

**Comment by icecube-trac** [comment]
Trac update at 20150921T18:46:58: 
* kkrings changed resolution from "" to "fixed"
* kkrings changed status from "accepted" to "closed"
* kkrings commented:

> In changeset 137805:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="137805"
> Solved deprecation for IPython 4 by doing an explicit version check; fixes #1351.
> ```
